src/index.css

```diff
@@ -1,7 +1,78 @@
+@import url('https://fonts.googleapis.com/css2?family=Raleway:wght@400;500;600;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Mulish:wght@400;500;600;700&display=swap');
+
+:root {
+  /* FONTS */
+
+  --Raleway: 'Raleway', sans-serif;
+  --Mulish: 'Mulish', sans-serif;
+
+  /* SIZES  */
+
+  --fs-12: 1.2rem;
+  --fs-14: 1.4rem;
+  --fs-16: 1.6rem;
+  --fs-20: 2rem;
+  --fs-24: 2.4rem;
+  --fs-26: 2.6rem;
+  --fs-40: 4rem;
```

Comment:
it would be better to use small/medium/large/h1/h2/h3/h4 sizes instead of transforming it to px :)